File: What century is it?.js
const whatCentury = (year) => {
  const result = String(Math.ceil((parseInt(year) / 1000) * 10) / 10 * 10);
  if (result[result.length - 2] === '1') {
    return result + "th";
  } else if (result[result.length - 1] === '1') {
      return result + "st";
  } else if (result[result.length - 1] === '2') {
      return result + "nd";
  } else if (result[result.length - 1] === '3') {
      return result + "rd";
  } else {
      return result + "th";
  }
}

File: What century is it?.py
import math
def what_century(year):
    result = str(int((math.ceil(int(year)/1000 * 10)/10)*10))
    if result[-2]=='1':
        return result+"th"
    elif result[-1]=='1':
        return result+"st"
    elif result[-1]=='2':
        return result+"nd"
    elif result[-1]=='3':
        return result+"rd"
    else:
        return result+"th"
